### PR TITLE
Lego 3489 oppdatere typografi

### PR DESCRIPTION
**Pull request by majabnils**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3489

### CODE REVIEW

File: packages/elvis/CHANGELOG.json [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+      "version": "19.0.0",
+      "date": "May 31, 2024",
+      "changelog": [
+        {
+          "type": "breaking_changes",
+          "changes": [
+            "Updated internal <code>@elvia/elvis-typography</code> dependency to <code>^3.0.0</code>"
+          ],
+          "components": [{ "displayName": "Typography", "url": "https://design.elvia.io/brand/typography" }]
+        }
```
Comment (TrygveUrdahl) [inline packages/elvis/CHANGELOG.json:15]: I don't think this helps our users understand what's changed. I would instead make what you've written here to be a patch, and add a breaking_change entry and describe pretty much the same as what you did in the elvis-typography changelog. 
If there are any changes that our users need to do, add a "fixes"-entry too 😄

File: packages/elvis-typography/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Elvia Typography Changelog
 
+## 3.0.0 (31.05.24)
```
Comment (TomSchrier) [inline packages/elvis-typography/CHANGELOG.md:3]: Remember to update the date before merge

File: packages/elvis/CHANGELOG.json [hunk 3]
```diff
@@ -2,6 +2,27 @@
   "$schema": "../components/changelogSchema.json",
   "name": "elvis",
   "content": [
+    {
+      "version": "19.0.0",
+      "date": "June 6, 2024",
```
Comment (TomSchrier) [inline packages/elvis/CHANGELOG.json:7]: Remember to update the date before merge here too

Also, remember to include the changes of https://github.com/3lvia/designsystem/pull/2917 and https://github.com/3lvia/designsystem/pull/2911

**Review by TomSchrier — APPROVED** [review]
LGTM

**Pull request merged**